airflow/cli/cli_parser.py

```diff
@@ -142,6 +142,10 @@ def positive_int(value):
     ("-e", "--end-date"),
     help="Override end_date YYYY-MM-DD",
     type=parsedate)
+ARG_OUTPUT_PATH = Arg(
+    ("-o", "--output-path",),
+    help="The output for generated yaml files",
```

Comment:
```suggestion
    help="The output for generated yaml files",
    type=argparse.FileType('w')
```
See: https://docs.python.org/3/library/argparse.html#filetype-objects

Reply:
This hasn't been fixed @dimberman